goal_src/jak2/engine/debug/editable.gc

```diff
@@ -5,5 +5,1749 @@
+        )
+      (add-debug-bound
+        324
+        (the-as transform gp-0)
+        sv-112
+        (the-as rgba (if (logtest? (-> obj flags) (editable-flag orient))
+                         (-> (new 'static 'array uint64 1 #x800000ff) 0)
+                         (-> (new 'static 'array uint64 1 #x8000ffff) 0)
+                         )
+                )
+        (the-as rgba (if (zero? (logand (-> obj flags) (editable-flag orient)))
+                         (-> (new 'static 'array uint64 1 #x800000ff) 0)
+                         (-> (new 'static 'array uint64 1 #x8000ffff) 0)
+                         )
+                )
+        (if (logtest? (-> obj flags) (editable-flag selected))
+            1
+            0
+            )
+        )
+      )
+    )
+  0
+  (none)
+  )
+
+(defmethod edit-get-trans editable-plane ((obj editable-plane))
+  "Returns the `trans` [[vector]] or [[*null-vector*]]"
+  (if (>= (-> obj length) 1)
+      (edit-get-trans (-> obj vertex 0))
+      *null-vector*
+      )
+  )
+
+(defmethod editable-method-22 editable-plane ((obj editable-plane) (arg0 editable-array) (arg1 int) (arg2 int))
+  (let ((s4-0 (clear *temp-string*)))
+    (format
+      s4-0
+      "insert into region_face set kind='plane',region_id=~D,radius=~f"
+      (-> obj region id)
+      (-> obj radius)
+      )
+    (let ((a0-4 (sql-query s4-0)))
+      (if (!= (-> a0-4 error) 'modify)
+          (return #f)
+          )
+      )
+    )
+  (dotimes (s4-1 (-> obj length))
+    (editable-method-22 (-> obj vertex s4-1) arg0 2 s4-1)
+    )
+  (logclear! (-> obj flags) (editable-flag changed))
+  #t
+  )
+
+(defmethod editable-method-25 editable-plane ((obj editable-plane) (arg0 editable-array))
+  (dotimes (s4-0 (-> obj length))
+    (let ((s3-0 (-> obj vertex s4-0)))
+      (set! (-> s3-0 owner) (delete! obj (-> s3-0 owner)))
+      )
+    )
+  ((the-as (function editable-plane editable-array none) (find-parent-method editable-plane 25)) obj arg0)
+  (none)
+  )
+
+(defmethod editable-method-26 editable-plane ((obj editable-plane) (arg0 editable) (arg1 editable-array))
+  (editable-array-method-15 arg1 obj)
+  ((method-of-type editable editable-method-26) obj arg0 arg1)
+  (none)
+  )
+
+(defmethod editable-method-27 editable-plane ((obj editable-plane) (arg0 editable-array))
+  (let ((gp-1
+          ((the-as (function editable-plane editable-array editable) (find-parent-method editable-plane 27)) obj arg0)
+          )
+        )
+    (dotimes (s4-0 (-> (the-as editable-plane gp-1) length))
+      (let ((v1-3 (editable-method-27 (-> (the-as editable-plane (+ (* s4-0 4) (the-as int gp-1))) vertex 0) arg0))
+            (a0-5 (+ (* s4-0 4) (the-as int gp-1)))
+            )
+        (s.w! (+ a0-5 28) v1-3)
+        )
+      (let ((v0-3 (cons gp-1 (-> (the-as editable-plane (+ (* s4-0 4) (the-as int gp-1))) vertex 0 owner)))
+            (v1-10 (l.wu (+ (* s4-0 4) (the-as int gp-1) 28)))
+            )
+        (s.w! (+ v1-10 16) v0-3)
```

Comment:
FYI these `s.w!`s and `l.wu`'s compile now, but we should eventually fix these (don't worry about it here, there's still other todos for this file so it can wait)